settings/prod.py

```diff
@@ -31,3 +32,6 @@
 STATICFILES_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
 S3_URL = 'http://%s.s3.amazonaws.com/' % AWS_STORAGE_BUCKET_NAME
 STATIC_URL = S3_URL
+MEDIAFILES_LOCATION = 'media'
+MEDIA_URL = 'http://%s.s3.amazonaws.com/%s/' % (AWS_STORAGE_BUCKET_NAME, MEDIAFILES_LOCATION)
+DEFAULT_FILE_STORAGE = 'custom_storages.MediaStorage'
```

Comment:
@RishabhJain2018 I think you forgot to add the custom_storages file. Can you add it?